bugbot/rules/file_crash_bug.py

```diff
@@ -86,6 +86,18 @@ def get_bugs(self, date):
             "Firefox", "nightly"
         )
         signatures = data_fetcher.analyze()
+        # This is the last filtering stage which aims to avoid filling bugs for
+        # junky crashes, where the volume is low and the crashes do not show
+        # signals of being actionable or critical.
+        signatures = [
+            signature
+            for signature in signatures
+            if signature.num_installs > 20
+            or signature.num_crashes > 200
```

Comment:
It kind of feels like there should be some "and" in here instead of just "or" but I'm not entirely sure how that should work. For instance, we probably don't want a bug filed if 1 install is crashing 300 times (which does happen...). Maybe just drop num_crashes from here entirely? I'm not sure.

Reply:
Crashes with only one installation are already filtered out in the first step (that excludes a lot of signatures).